functions/Set-DbaNetworkConfiguration.ps1

```diff
@@ -0,0 +1,413 @@
+function Set-DbaNetworkConfiguration {
+    <#
+    .SYNOPSIS
+        Sets the network configuration of a SQL Server instance.
+
+    .DESCRIPTION
+        Sets the network configuration of a SQL Server instance.
+
+        Parameters are available for typical tasks like enabling or disabling a protokoll or switching between dynamic and static ports.
+        To be more flexible, you can use Get-DbaNetworkConfiguration to get an object that represents the complete network configuration
+        like you would see it with the SQL Server Configuration Manager.
+        You can change any setting of this object and then pass it to Set-DbaNetworkConfiguration via pipeline or InputObject parameter.
+
+        Every change to the network configuration needs a service restart to take effect. To do this, use the RestartService parameter.
```

Comment:
```suggestion
        A change to the network configuration with SQL Server requires a restart to take effect, support for this can be done via the RestartService parameter.
```